addon/ng2/blueprints/ng2/files/e2e/app.e2e.ts

```diff
@@ -1,4 +1,5 @@
-import "angular2/testing";
+/// <reference path="../typings/main.d.ts" />
```

Comment:
Could you do the same as the main project and add a `typings.d.ts` file to this directory that includes this line. Include that new file in the build.